src/client/mod.rs

```diff
@@ -199,6 +203,10 @@ impl Default for ClientConfig {
             conn_init_thread_num: 2,
             query_concurrency_limit: None,
 
+            conn_reader_threads: 6,
+            conn_writer_threads: 4,
+            default_threads_num: 2,
```

Comment:
Why not keep them in the same style? And how about setting all these runtime's thread number is 2 by default?
```suggestion
            conn_reader_thread_num: 2,
            conn_writer_thread_num: 2,
            default_threads_num: 2,
```